tests/foreman/ui/test_activationkey.py

```diff
@@ -1144,7 +1145,7 @@ def test_positive_delete_manifest(session, function_entitlement_manifest_org, ta
 @pytest.mark.skip_if_not_set('clients')
 @pytest.mark.tier3
 @pytest.mark.skipif((not settings.robottelo.repos_hosting_url), reason='Missing repos_hosting_url')
-def test_positive_ak_with_custom_product_on_rhel6(session, rhel6_contenthost, target_sat):
+def test_positive_ak_with_custom_product_on_rhel6(session, rhel_contenthost, target_sat):
```

Comment:
As you're replacing rhel6_contenthost with rhel_contenthost, and this test looks specific to rhel6, so can you add missing `rhel_ver_match` or `rhel_ver_list` marks on the test for el6?

Reply:
Yes. I missed adding it. Thanks for pointing.